daemon/openrazer_daemon/dbus_services/dbus_methods/mambaelite.py

```diff
@@ -0,0 +1,534 @@
+from openrazer_daemon.dbus_services import endpoint
+from openrazer_daemon.dbus_services.dbus_methods.lanceheadte import get_left_brightness as _get_left_brightness, \
+    get_right_brightness as _get_right_brightness, \
+    set_left_wave as _set_left_wave, set_right_wave as _set_right_wave, \
+    set_left_static as _set_left_static, set_right_static as _set_right_static, \
+    set_left_spectrum as _set_left_spectrum, set_right_spectrum as _set_right_spectrum, \
+    set_left_none as _set_left_none, set_right_none as _set_right_none, \
+    set_left_reactive as _set_left_reactive, set_right_reactive as _set_right_reactive, \
+    set_left_breath_random as _set_left_breath_random, set_right_breath_random as _set_right_breath_random, \
+    set_left_breath_single as _set_left_breath_single, set_right_breath_single as _set_right_breath_single, \
+    set_left_breath_dual as _set_left_breath_dual, set_right_breath_dual as _set_right_breath_dual
+
+
+@endpoint('razer.device.lighting.logo', 'setLogoStatic', in_sig='yyy')
+def set_logo_static_mamba_elite(self, red, green, blue):
```

Comment:
These methods seem to be exactly the same as in `dbus_services/dbus_methods/nagahexv2.py` , why are they duplicated? (if you got confused by the nagahexv2 name, just ignore that. The naming conventions in this project are very legacy-driven :wink: )

Reply:
OK, so I will fix